scripts/.bashrc_patch

```diff
@@ -1,3 +1,9 @@
-if [[ -z "${NO_SETUP}" ]]; then
-  source entry.sh
+export ENTITIES="/home/node/ardrive-cli/.bats/entities.sh"
```

Comment:
Consider renaming to `ENTITIES_PATH`

Reply:
Also, I'm not sure where to look for that `.sh` file

Reply:
Maybe ENTITIES_SCRIPT?

Reply:
Yes, maybe that's fine. We ain't using that suffix anywhere else :+1:

Reply:
Lets keep it then